code_schemes/BRA_shareable.json

```diff
@@ -0,0 +1,17 @@
+{
+  "SchemeID": "Scheme-58781eb2",
```

Comment:
We should probably just add the control code to the main scheme placeholder in the future.

Reply:
Yeah, that make more sense if we knew we were going to do this in advance, provided we're only sharing with one other organisation/our definition of "safe to share" is the same for all the organisations we're sharing with.